### ISSUE TITLE
v4.4.3 Loses Transitions and Timeout Property so Notifications cannot be dismissed

### ISSUE THREAD

**Issue by gtwilliams03** [chadly/react-bs-notifier#41]
I am upgraded to v4.4.3 using a simple `AlertList` that has worked for a while (and works up until v4.4.2):

```
      <div>
        <AlertList
            position="top-right"
            alerts={this.props.notifications}
            timeout={4000}
            onDismiss={notification => this.props.removeNotification(notification)}
        />
      </div>
```

Now, at v4.4.3, the alerts appear, but cannot be dismissed with the `x` button or timeout.  

Was there a breaking change in v4.4.3?  Thank you for your continued efforts on this project. It is great.  Period.

**Comment by gtwilliams03** [comment]
I rolled back to `v4.4.2` and it works fine.

**Comment by idearobin** [comment]
Also experiencing this. Great work thus far!

**Comment by chadly** [comment]
Fixed in `v4.4.4`
